src/pages/profile/components/Sidebar.tsx

```diff
@@ -124,7 +124,7 @@ export default function Sidebar({
     Object.keys(updatedData).forEach((key: string) => {
       const typedKey = key as keyof User
 
-      if (updatedData[typedKey] && originalData[typedKey] !== updatedData[typedKey]) {
+      if (typeof updatedData[typedKey] === 'string' && originalData[typedKey] !== updatedData[typedKey]) {
```

Comment:
Can we use `isInvalidInput` function here and few other files?

Reply:
Yes, we can use this function if we find suitable place to use it.

Here in this case simply checking if its a string will work for us as we are only updating the keys of user profile which are all string value at the moment.

Reply:
Agree with @kranthicodes here 😊 
It's easier to find all places where we are type checking if we re-use `isInvalidInput` everywhere